q8.py

```diff
@@ -6,5 +6,14 @@
 
 (This should print out 18 words in total)
 """
+import random
 
-# YOUR CODE HERE
\ No newline at end of file
+def random_word():
+    first_letters = ["c", "t", "b"]
+    middle_letters = ["a", "o"]
+    last_letters = ["p", "t", "n"]
+    return f"{random.choice(first_letters)}{random.choice(middle_letters)}{random.choice(last_letters)}"
+
+if __name__ == "__main__":
+  for _ in range(18):
+    print(random_word())
```

Comment:
the goal was to print out all unique words, not a random word. but good job with printing out a random word!